meta: provide example activities

Trott opened the pull request:
<!--
Thank you for your pull request. Please review below requirements.

Bug fixes and new features should include tests and possibly benchmarks.

Contributors guide: https://github.com/nodejs/node/blob/master/CONTRIBUTING.md
-->

##### Checklist
<!-- Remove items that do not apply. For completed items, change [ ] to [x]. -->

- [x] documentation is changed or added
- [x] commit message follows commit guidelines

##### Affected core subsystem(s)
<!-- Provide affected core subsystem(s) (like doc, cluster, crypto, etc). -->
meta

##### Description of change
<!-- Provide a description of the change below this comment. -->

Provide example activities to better distinguish Collaborator activities
from CTC member activities.

@nodejs/ctc

misterdjules commented:
This definitely looks like an improvement. I think it would be even better if the changes in github.com/nodejs/node/pull/7720 referred to the "Typical activities of a CTC member" paragraph added in this PR. Thank you for doing this!

thefourtheye commented on GOVERNANCE.md:
```diff
@@ -61,6 +61,14 @@ For the current list of Collaborators, see the project
+
+* contributing code and documentation changes to improve the project
+* merging pull requests
+* commenting on issues and pull requests
```
Would reviewing be better instead of commenting?

Trott replied:
Possibly, although `reviewing` doesn't necessarily entail leaving an artifact of the review, whereas `commenting` does. I'd be fine with either word.

jasnell replied:
`reviewing and commenting`

Trott replied:
@thefourtheye @jasnell OK, changed it from `commenting` to `reviewing and commenting`

Fishrock123 commented:
Hmmm, I'm a bit skeptical of this.

I have asserted quite in the past there is bare-minimal requirements for CTC and collaborators, particularly those who are not paid to work on Node.

People doing this for free should not feel obligated to give time when it isn't feasible.

So for collaborators, I've always said something along the lines of "only do what you can, this shouldn't be burdensome". For CTC members: "You're only required to attend meetings, it'd be cool if you'd do other stuff, but don't feel more pressured to".

Trott commented:
@Fishrock123 None of these are listed as *requirements*, just typical activities. This is an attempt to answer the questions:

* What is a Collaborator? What does a Collaborator do?
* What does a CTC member do that a Collaborator doesn't?

rvagg commented:
@trott how about adding "(these are not requirements for membership of the Collaborators group)" to the lead-in to the first list would help with @Fishrock123's concerns?

In the second list, how about something like "facilitate the smooth day-to-day running of the Node.js project", to reinforce that the CTC is supposed to be the oil, or grease, that keeps the machine going.

Trott commented:
@Fishrock123 @rvagg I've added this paragraph to hopefully allay the concerns raised:

> While the above are typical things done by Collaborators, there are no required
activities to retain Collaborator status. There is currently no process by which
inactive Collaborators are removed from the project.

@rvagg I also added your suggestion (slightly modified) for the CTC list. My only hesitation is that it is not specific in the least and the reason to introduce these lists was to give concrete examples of CTC activities vs. Collaborator activities.

@misterdjules Yes, I agree, and I will update that PR to refer to these items if this PR lands first (which I expect it will).

Fishrock123 commented:
seems good to me

bnoordhuis commented:
Lukewarm LGTM.  I suppose you might as well land it now that you've written it but I don't think it's hugely valuable.

The distinguishing characteristic of a CTC member versus a regular collaborator is that the former has decision power and the moral obligations that come with that.  That's really all there is to it, IMO: contentious issues are decided by the CTC.

Trott commented:
I guess I'd like to get the @nodejs/ctc temperature on this one. Like, choose one:

* LGTM! Ship it!
* -1. I don't like this.
* Eh, whatever, I don't really care either way to be honest. Stop bugging me with notifications, please.

misterdjules commented:
LGTM.

Trott commented:
Trying to wrap this up one way or another, so I'm going to go ahead and land this sometime after another 24 hours unless someone on @nodejs/ctc indicates that they would prefer that it not land.

Trott commented:
I'm labeling this `ctc-agenda` so that we can hopefully make a very quick decision as to whether or not to land it. I'm fine with landing it, and I'm fine with closing it without landing. I just don't want to leave it hanging open forever.

Normally, I'd take an LGTM or two as sufficient grounds for landing something absent an objection from anyone. But I'm inclined to be a lot more cautious with documenting CTC rules and related governance material.

misterdjules commented:
@Trott I haven't seen any objection after the latest updates, and the CTC meeting's agenda seems really packed this week. Is it possible for other @nodejs/ctc members to weigh in today and only keep this on the CTC meeting's agenda if there are actually objections?

jasnell commented on GOVERNANCE.md:
```diff
@@ -56,6 +56,18 @@ For the current list of Collaborators, see the project
+* merging pull requests
+* reviewing and commenting on issues and pull requests
+* participation in working groups
+* helping users and novice contributors
```
Would like to see these reordered a bit by priority, with merging pull requests being the last in the list.. perhaps...

* Helping users and novice contributors
* Contributing code and documentation changes...
* Reviewing and commenting on issues and pull requests
* Participation in working groups
* Merging pull requests

Trott replied:
@jasnell I've re-ordered the list per your suggestion.

Trott commented:
Per CTC meeting yesterday, I'm going to land this at this time. As with the quorum material, we can revise it or remove it if anyone objects subsequently.

Trott commented:
Landed in 99e2d4380120d3aad0cd4dafc59b83be9f970256
